src/pid.cpp

```diff
@@ -126,6 +126,9 @@ void Pid::setGains(double p, double i, double d, double i_max, double i_min, boo
 
 void Pid::setGains(const Gains & gains)
 {
+  if (gains.i_min_ > gains.i_max_) {
```

Comment:
Should we also check this in the line 120?

Reply:
we could, but in the end `void Pid::setGains(const Gains & gains)` is called anyways.